scraper.py

```diff
@@ -92,24 +144,23 @@ def make_soup(url, encoding, absolute=False, base='', parser='lxml'):
     return soup
 
 
-def save(url, date, title, explanation, picture_url, data_version):
-    data = {
-        'url': url,
-        'date': date,
-        'title': title,
-        'explanation': explanation,
-        'picture_url': picture_url,
-    }
-
-    version_data = {
-        'url': url,
-        'data_version': data_version,
-    }
+def save(url, date, title, credit, explanation, picture_thumbnail_url, picture_url, video_url, data_version):
+    data = OrderedDict()
```

Comment:
There's no real advantage apart from just making the database (as well as any data returned from queries) look nicer. :smile: As you can see in my database, the columns are all in the order I specified (starting with `url`).